contents/docs/apps/advanced-geoip.md

```diff
@@ -19,24 +19,49 @@ Not running 1.30.0? Find out [how to update your self-hosted PostHog deployment]
 
 You'll also need to have [the GeoIP Entricher app](/apps/geoip-enrichment) installed. 
 
-### How do I install the Customer.io Connector app?
+### How do I install this app?
 
-1. Visit the 'Apps' page in your instance of PostHog.
-2. Search for 'Advanced GeoIP' and select the app, press Install.
+1. Make sure you have enabled `opt_in_site_apps: true` in your posthog-js config.
+2. Install the app from the PostHog App Repository
+3. Customize the text and theme using the app config
```

Comment:
They need the forth step of creating a feature flag to starting inviting people to interview